apps/api/docs/production-setup.md

```diff
@@ -52,21 +52,21 @@ Make sure Redis and MariaDB server are up and running.
   MAILGUN_API_KEY=your-mailgun-api-key
   MAILGUN_HOST=api.mailgun.net
   MAILGUN_DOMAIN=your.mailgun.domain
-  
+
   # WhatsApp 360Dialog
   ENABLE_WA360DIALOG=true
   WA_360_DIALOG_URL=https://waba.360dialog.io/v1/messages
   WA_360_DIALOG_API_KEY=
 
   # TEMP
-  APP_NAME=osmo_notify
+  APP_NAME=osmoX
```

Comment:
Update to `OsmoX`

Reply:
Addressed in https://github.com/OsmosysSoftware/osmo-x/pull/152/commits/ffca8627cb4ae9663e16c6d1288440193d1ada1b